### PR TITLE
moderation added (prewiew only)

### PR DESCRIPTION
**Pull request by OleksandrZhelihovskyi**
moderation added (prewiew only)

### CODE REVIEW

File: controllers/moderationController.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+const { CheaterPhone, PhoneDescription } = require('../models');
+
+// READ (all categories)
```
Comment (VitaliyBogdanuk) [inline controllers/moderationController.js:3]: all categories?

File: views/pages/moderation.ejs [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+    <%- include('../templates/navbar', {breadcrumbs: 'Moderation' }); -%>
+    <!-- End Navbar -->
+    <div class="container-fluid py-4">
+        <div class="row">
+            <div class="col-12">
+                <div class="card my-4">
+                    <div class="card-header p-0 position-relative mt-n4 mx-3 z-index-2">
+                        <div class="d-flex justify-content-lg-between bg-gradient-primary shadow-primary border-radius-lg pt-4 pb-3">
+                            <h6 class="text-white text-capitalize ps-3">Moderation</h6>
+                          <!-- <a href="/forms/category" class="text-white px-3"><i class="fa fa-plus p-1"></i>Add category</a>-->
```
Comment (VitaliyBogdanuk) [inline views/pages/moderation.ejs:14]: ?

**Review by VitaliyBogdanuk — CHANGES_REQUESTED** [review]
@OleksandrZhelihovskyi

File: controllers/moderationController.js (hunk 1 above)
Comment (OleksandrZhelihovskyi) [reply controllers/moderationController.js:3]: lol, nope) phones to moderate ofc) fixed

File: views/pages/moderation.ejs (hunk 2 above)
Comment (OleksandrZhelihovskyi) [reply views/pages/moderation.ejs:14]: deleted

**Pull request merged**